DiscordBot/Program.cs

```diff
@@ -47,102 +45,75 @@ private static async Task MainAsync()
 
             commands.RegisterCommands<EventCommands>();
 
+            var reactionBuffer = new ReactionBuffer(eventsSheetsService);
+
             discord.UseInteractivity(new InteractivityConfiguration());
             discord.MessageReactionAdded += async (client, eventArguments) =>
             {
-                await OnMessageReactionAdded(client, eventArguments, eventsSheetsService);
+                await OnMessageReactionAdded(client, eventArguments, eventsSheetsService, reactionBuffer);
             };
 
             await discord.ConnectAsync();
             await Task.Delay(-1);
         }
 
-        private static async Task OnMessageReactionAdded(
+        private static Task OnMessageReactionAdded(
```

Comment:
Does this need to return a Task at all, or could it just have void return type? 🙂

Reply:
I've just had a look at this with @kwunkiuma. Basically, the `discord.MessageReactionAdded` event handler on line 51 expects to receive a `Task`, but there's no need for `OnMessageReactionAdded` to returning one in all cases. I've changed it so it returns `void` and the hander here returns a `Task.CompletedTask` in 8c81656 🙂